src/App.jsx

```diff
@@ -14,70 +17,82 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [title, setTitle] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const handleTitleChange = (good) => {
+    if (good === title) {
+      setTitle(null);
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      return;
+    }
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+    setTitle(good);
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {title
+          ? `${title} is selected`
+          : 'No goods selected'
+        }
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+        {title
+          && (
             <button
-              data-cy="RemoveButton"
+              data-cy="ClearButton"
               type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+              className="delete ml-3"
+              aria-label="clearTitle"
+              onClick={() => handleTitleChange(null)}
+            />
+          )}
+      </h1>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
+              key={uuidv4()}
+              className={
+                classes({ 'has-background-success-light': good === title })
+              }
             >
-              +
-            </button>
-          </td>
+              <td>
+                <button
+                  data-cy={good === title
```

Comment:
create variable like `isSelectedGood` for `good === title`, don't repeat yourself

Reply:
Thanks, good idea, did it.